packages/components/molecules/f-navigation-links/src/components/_tests/__snapshots__/NavigationLinks.test.js.snap

```diff
@@ -2,43 +2,43 @@
 
 exports[`NavigationLinks should display the list elements correctly for each \`link\` supplied 1`] = `
 <nav aria-label="navigation" data-test-id="navigation-links-component">
-  <ul class="c-navigationLinks">
```

Comment:
I've had to update the snapshots because css compilation when turned off can't pick these up. The compilation has been turned off because `node-sass` has been removed.